app/bundles/CoreBundle/Helper/PathsHelper.php

```diff
@@ -100,6 +106,8 @@ public function getSystemPath($name, $fullPath = false)
             if (substr($path, -1) === '/') {
                 $path = substr($path, 0, -1);
             }
+        } elseif ($name == 'assets') {
+            return realpath($this->assetPath);
```

Comment:
I found an issue here (what we talked about on Slack).

> Warning: fopen(D:\git\mautic/D:\git\mautic/css/libraries.css): failed to open stream: Invalid argument in D:\git\mautic\app\bundles\CoreBundle\Helper\AssetGenerationHelper.php on line 141

You should replace line 110 by
```
if ($this->assetPath == null){  
    return '';  
} else {  
    return realpath($this->assetPath);  
} 
```